.travis.yml

```diff
@@ -29,6 +29,18 @@ matrix:
       env:
         - BACKEND=Intel10_64lp
         - BUILD_STATIC=ON
+        - BUILD_PACKAGE=ON
```

Comment:
you need to disable the STATIC that does not use matio as otherwise you have the fPIC problem:

https://travis-ci.org/massich/findmkl_openmeeg/jobs/352541797